DQM/TrackingMonitorSource/python/TrackingSourceConfig_Tier0_cff.py

```diff
@@ -178,7 +178,7 @@
     label = 'TrackerCollisionSelectedTrackMonCommon' + str(tracks)
     TrackingDQMSourceTier0 += locals()[label]
 # seeding monitoring
-for era in _cfg.allEras():
+for era in _cfg.allEras() + ["trackingPhase2PU140"]: # FIXME:: allEras() extension should be removed when phase2 tracking is migrated to eras
     postfix = _cfg.postfix(era)
```

Comment:
The issue in 14649 is corrected in 14713
________________________________
From: Slava Krutelyov [notifications@github.com]
Sent: 01 June 2016 18:06
To: cms-sw/cmssw
Cc: Sunanda Banerjee; Mention
Subject: Re: [cms-sw/cmssw] Phase2 tracking validation in 81X (#14735)


In DQM/TrackingMonitorSource/python/TrackingSourceConfig_Tier0_cff.py<https://github.com/cms-sw/cmssw/pull/14735#discussion_r65391489>:

> @@ -178,7 +178,7 @@
>      label = 'TrackerCollisionSelectedTrackMonCommon' + str(tracks)
>      TrackingDQMSourceTier0 += locals()[label]
>  # seeding monitoring
> -for era in _cfg.allEras():
> +for era in _cfg.allEras() + ["trackingPhase2PU140"]: # FIXME:: allEras() extension should be removed when phase2 tracking is migrated to eras
>      postfix = _cfg.postfix(era)


... no, apparently not. The name is a bit misleading.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/cms-sw/cmssw/pull/14735/files/1fd5606758b7209ae4d113555caa8c633e811e14#r65391489>, or mute the thread<https://github.com/notifications/unsubscribe/AEzMugSQUHPA0ceeW5N2AM82pslcC4ffks5qHa4XgaJpZM4Irta4>.